docs/apache-airflow/tutorial_taskflow_api.rst

```diff
@@ -301,7 +301,7 @@ to a TaskFlow function which parses the response as JSON.
     )
 
 
-    @task(max_retries=2)
```

Comment:
I think the point here is to show that you can pass "operator" params to the task decorator, separately from the function arguments.

Reply:
Agreed but I'm not sure it's adding anything in this section.

However, this has come up many times on Slack, StackOverflow, Discussions, etc., but I feel like a mini chapter on this in the tutorial is a good addition instead. (Something on my list I need to get to.) That way we can be really explicit about the functionality in TaskFlow.

Maybe even some TaskFlow "recipes" documentation.